packages/protocol/migrationsConfig.js

```diff
@@ -69,8 +70,7 @@ const DefaultConfig = {
     spread: 5 / 1000,
     reserveFraction: 1 / 100,
     updateFrequency: 5 * MINUTE, // 5 minutes
-    // TODO(asa): This is too low
-    minimumReports: 1,
+    minimumReports: 10,
```

Comment:
Still discussing this in the oracles bubble but likely needs to be smaller than 10
```suggestion
    minimumReports: 5,
```